src/input/espresso.rs

```diff
@@ -520,7 +625,11 @@ impl EpochState {
             ))
         })?;
 
-        Ok(Self::new(epoch, stake_table, drb_result))
+        let mut state = Self::new(epoch, stake_table, drb_result);
+        let block_reward = espresso.block_reward(epoch).await?;
+        state.block_reward = Some(block_reward);
```

Comment:
Since we unconditionally set `block_reward` to `Some` here, when would it ever be `None`? Can we just remove the `Option<>`?